internal-packages/workflow-designer-ui/src/editor/v2/v2-placeholder.tsx

```diff
@@ -6,7 +6,6 @@ import {
 } from "@giselle-sdk/giselle/react";
 import { useCallback, useState } from "react";
 import { ReadOnlyBanner } from "../../ui/read-only-banner";
-import { KeyboardShortcuts } from "../keyboard-shortcuts";
```

Comment:
:memo: Removed along with the deletion of internal-packages/workflow-designer-ui/src/editor/keyboard-shortcuts.tsx